truss/cli/cli.py

```diff
@@ -449,6 +456,7 @@ def deploy(
             promote=promote,
             publish=publish,
             only_generate_trusses=dryrun,
+            user_env=json.loads(user_env) if user_env else {},
```

Comment:
Let's make this return a nice error message (i'd catch this and return something more readable) if the json doesn't parse.